### PR TITLE
Database lesson3

### PR DESCRIPTION
**Pull request by maccrin**
This PR for DB week3 homework

### CODE REVIEW

File: databases/week3/table_creation/table_creation.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+CREATE TABLE  `review` (
+  `id` INT (10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `title` VARCHAR(100) NOT NULL,
+  `description` TEXT(500) NULL,
+  `meal_id` INT UNSIGNED NOT NULL,
+  `stars` INT UNSIGNED NULL,
+  `created_date` DATE NULL,
+  CONSTRAINT `fk_review_meal`
+    FOREIGN KEY (`meal_id`)  REFERENCES `meal` (`id`)ON DELETE CASCADE ON UPDATE CASCADE)
+ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```
Comment (Elia-Git) [inline databases/week3/table_creation/table_creation.sql:35]: cool

File: databases/week3/additional_queries/additional_queries.sql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#Get meals that has a price smaller than a specific price fx 90
+select * from meal where price< 200;
+#Get meals that still has available reservations;
+
+#Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde
+select * from meal where title like'%paneer%';
+#Get meals that has been created between two dates
+select * from meal where created_date  BETWEEN '2023-01-20' AND '2023-01-30';
+#Get only specific number of meals fx return only 5 meals
+select * from meal limit 3;
```
Comment (Elia-Git) [inline databases/week3/additional_queries/additional_queries.sql:24]: Nothing wrong. Use` * `only if necessary otherwise better to sort with few columns.

**Review by Elia-Git — APPROVED** [review]
Ship it pls.

File: databases/week3/additional_queries/additional_queries.sql (hunk 2 above)
Comment (maccrin) [reply databases/week3/additional_queries/additional_queries.sql:24]: 👍🏻

**Pull request merged**